### ISSUE TITLE
English version/german version

### ISSUE THREAD

**Issue by andymasoner** [stringparser/artist-carpenter-berlin#13]
Make a section section for german version of site - I can translate over time

**Comment by stringparser** [comment]
Yeah, this is a common practice. We can do this once all the english text is done and then add a language switch menu.
